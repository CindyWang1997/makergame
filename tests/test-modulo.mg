using std;
namespace print = std::print;

object main {
  event create {
    print::i(5 % 2);
    print::i(-5 % 2);
    print::f(6 % 2.5);
    print::f(5.5 % 2);
    print::f(5.5 % 2.5);
    std::game::end();
  }
}
