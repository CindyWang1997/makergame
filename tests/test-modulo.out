1
-1
1.000000
1.500000
0.500000
